Earthfile

```diff
@@ -34,6 +34,13 @@ install-deps:
     RUN sudo apt-get install nodejs -y
     RUN npm install --global yarn
     RUN npm install --global openapi-typescript-codegen
+    RUN apt install unzip -y
```

Comment:
I think there were some other lines that install rpk and unzip in the Earthfile when we looked at it, so maybe we can remove those since they are now here in the "super-task"?